.circleci/config.yml

```diff
@@ -115,8 +115,7 @@ jobs:
       - setup_remote_docker
       - heroku-buildpacks/install-build-dependencies
       - run:
-          name: cargo make test
-          command: cargo make test
+          command: cargo fmt --all -- --check
```

Comment:
Thank you for fixing this - I saw it in the past but never got around to fixing it.

(For a later PR) CI also doesn't run clippy at present - would be idea. to add it. Also the existing `cargo test` entry in this file doesn't use `--locked`. For best practice options (at least from my research back then), see:
https://github.com/heroku/heroku-buildpacks-repo-template/blob/main/.circleci/config.yml